### PR TITLE
Backup etcd

### PR DESCRIPTION
**Pull request by Dexus**

### CODE REVIEW

File: openstack/backup-etcd/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          - name: OS_USERNAME
+            value: {{.Values.backup_os_username}}
+          - name: OS_USER_DOMAIN_NAME
+            value: {{.Values.backup_os_user_domain}}
+          - name: OS_PROJECT_NAME
+            value: {{.Values.backup_os_project_name}}
+          - name: OS_PROJECT_DOMAIN_NAME
+            value: {{.Values.backup_os_project_domain}}
+          - name: OS_REGION_NAME
+            value: {{.Values.backup_os_region_name}}
```
Comment (reimannf) [inline openstack/backup-etcd/templates/deployment.yaml]: Please switch to {{.Values.global.region}}

File: openstack/backup-etcd/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,73 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: backup-etcd
+  namespace: backup
+spec:
+  replicas: 1
```
Comment (reimannf) [inline openstack/backup-etcd/templates/deployment.yaml]: What happens if there is a helm deployment during a backup run? The pod will be killed, does this leaves strange artifacts or unfinished swift uplaods? Do we need a rollingUpdate strategy with `maxUnavailable: 0` or so?

File: openstack/backup-etcd/values.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+# this value are seconds 1800 are 30 minutes = 30 minutes * 60 seconds
+backup_interval: 1800
+
+# backup storage settings for swift
+backup_os_auth_url: DEFINED-IN-REGION-SECRETS
+backup_os_username: db_backup
+backup_os_user_domain: Default
+backup_os_project_name: master
+backup_os_project_domain: ccadmin
+backup_os_region_name: DEFINED-IN-REGION-SECRETS
```
Comment (reimannf) [inline openstack/backup-etcd/values.yaml]: That can be deleted now

**Comment by Dexus** [comment]
done

**Pull request merged**